release-please-config.json

```
@@ -0,0 +1,5 @@
+{
```

```suggestion
{
  "bootstrap-sha": "fc9873ed5e1baf428b6cb62c8af99b3ad8957c8a",
```

oops, forgot this. I means release-please will ignore any prior commits. It's a one-time setting that is ignored after the first Release PR is merged and can be safely removed at that point.